vllm/model_executor/layers/quantization/fp8.py

```diff
@@ -290,23 +290,29 @@ def create_weights(self, layer: Module, num_experts: int, hidden_size: int,
         # WEIGHT_SCALES
         # Allocate 2 scales for w1 and w3 respectively.
         # They will be combined to a single scale after weight loading.
-        w13_scale = torch.nn.Parameter(torch.ones(num_experts,
-                                                  2,
-                                                  dtype=torch.float32),
-                                       requires_grad=False)
-        layer.register_parameter("w13_scale", w13_scale)
+        w13_weight_scale = torch.nn.Parameter(torch.ones(num_experts,
+                                                         2,
+                                                         dtype=torch.float32),
+                                              requires_grad=False)
+        layer.register_parameter("w13_weight_scale", w13_weight_scale)
```

Comment:
Does that mean we have to update checkpoints after this PR?

Reply:
like checkpoints on disk? no. the layer has a mapping to take care of the name change